docs/helping.md

```diff
@@ -0,0 +1,142 @@
+# Helping
+
+This section is meant mostly for those that will be actively helping others in the Northstar Discord server or otherwise. If you've somehow stumbled upon this page trying to fix an issue for yourself, you should try checking the [troubleshooting page](installing-northstar/troubleshooting.md). If you're a helper, it also isn't a bad idea to look at this page a few times and try to have a rough idea of the things that are on it.
```

Comment:
I'd say have line breaks between sentences in the markdown source. Those don't get rendered as line breaks on GitBook (cause that's markdown convention) but make it easier to suggest changes to a single sentence cause it results in a smaller diff (sentence vs paragraph).

In other words

```suggestion
This section is meant mostly for those that will be actively helping others in the Northstar Discord server or otherwise.
If you've somehow stumbled upon this page trying to fix an issue for yourself, you should try checking the [troubleshooting page](installing-northstar/troubleshooting.md).
If you're a helper, it also isn't a bad idea to look at this page a few times and try to have a rough idea of the things that are on it.
```

This isn't the only place, just the first I found ^^